index.html

```diff
@@ -0,0 +1,1610 @@
+                </path>
+              </g>
+            </svg>
+          </a>
+          </div>
+          <div class="header-buttons col-7">
+            <a class="login text-white" href=" #">LOGIN</a>
+            <a class="sub-button " id="subscribe-btn" href=" #">
+              SUBSCRIBE
+            </a>
+            <button class="navbar-toggler p-0 ml-2" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
+              <span class="navbar-toggler-icon pt-2" ></span>
+            </button>
+           <label class="menu-bar" type="button" data-toggle="collapse"> <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+              width="24" height="24"
+              viewBox="0 0 172 172"
+              style=" fill:#000000;"><g fill="none" fill-rule="nonzero" stroke="none" 
+              stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10"
+               stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none"
+                font-size="none" text-anchor="none" style="mix-blend-mode: normal">
+                <path d="M0,172v-172h172v172z" fill="none"></path><g fill="#ffffff">
+                  <path d="M21.5,35.83333c-2.58456,-0.03655 -4.98858,1.32136 -6.29153,
+                  3.55376c-1.30295,2.2324 -1.30295,4.99342 0,7.22582c1.30295,2.2324 3.70697,
+                  3.59031 6.29153,3.55376h129c2.58456,0.03655 4.98858,-1.32136 6.29153,
+                  -3.55376c1.30295,-2.2324 1.30295,-4.99342 0,-7.22582c-1.30295,
+                  -2.2324 -3.70697,-3.59031 -6.29153,-3.55376zM21.5,78.83333c-2.58456,
+                  -0.03655 -4.98858,1.32136 -6.29153,3.55376c-1.30295,2.2324 -1.30295,
+                  4.99342 0,7.22582c1.30295,2.2324 3.70697,3.59031 6.29153,3.55376h129c2.58456,
+                  0.03655 4.98858,-1.32136 6.29153,-3.55376c1.30295,-2.2324 1.30295,-4.99342 0,
+                  -7.22582c-1.30295,-2.2324 -3.70697,-3.59031 -6.29153,-3.55376zM21.5,121.83333c-2.58456,
+                  -0.03655 -4.98858,1.32136 -6.29153,3.55376c-1.30295,2.2324 -1.30295,4.99342 0,
+                  7.22582c1.30295,2.2324 3.70697,3.59031 6.29153,3.55376h129c2.58456,0.03655 4.98858,-1.32136 6.29153,-3.55376c1.30295,-2.2324 1.30295,-4.99342 0,-7.22582c-1.30295,-2.2324 -3.70697,-3.59031 -6.29153,-3.55376z"></path>
+              </g></g></svg></label>
+          </div>
+        </div>
+      </header>
+    -->
```

Comment:
Remove HTML commented code

Reply:
let me work on the changes you have requested and resubmit